org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/InMemoryWorkspaceTest.xtend

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2017 TypeFox GmbH (http://www.typefox.io) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.ide.tests.server
+
+import org.eclipse.lsp4j.DidOpenTextDocumentParams
+import org.eclipse.lsp4j.Position
+import org.eclipse.lsp4j.TextDocumentIdentifier
+import org.eclipse.lsp4j.TextDocumentItem
+import org.eclipse.lsp4j.TextDocumentPositionParams
+import org.junit.Test
+
+class InMemoryWorkspaceTest extends AbstractTestLangLanguageServerTest {
+	
+	@Test
+	def void testCompletion() {
+		initialize[rootUri = null]
+		val inmemoryUri = 'inmemory:/mydoc.testlang'
```

Comment:
Is it MUST to use `inmemory` scheme if a root URI is not provided? or it is is just an example